runtime/interpreter/interpreter.go

```diff
@@ -2475,9 +2654,16 @@ func init() {
 			continue
 		}
 
-		if _, ok := converterNames[numberType.String()]; !ok {
+		// todo use TypeID's here?
+		typeName := numberType.String()
```

Comment:
The number type's type IDs and string representation are the same, so there isn't really any difference